### PR TITLE
Someone sent me a kudos for my posted/shared article notification

### PR DESCRIPTION
**Pull request by mohamedHedi1990**

### CODE REVIEW

File: webapp/src/main/webapp/WEB-INF/notification/templates/web/postNewsNotificationPlugin.gtmpl [hunk 1]
```diff
@@ -31,6 +31,12 @@
           case "COMMENT MY SHARED NEWS":
             notificationDescription =  _ctx.appRes("news.notification.description.comment.mySharedNews",currentUser,CONTENT_TITLE,spaceName);
             break;
+          case "SEND KUDOS FOR MY POSTED NEWS":
+            notificationDescription =  _ctx.appRes("news.notification.description.sendKudos.myPostedNews",currentUser,CONTENT_TITLE,spaceName);
```
Comment (aelbenney) [inline webapp/src/main/webapp/WEB-INF/notification/templates/web/postNewsNotificationPlugin.gtmpl]: I suggest adding the kudos icon in the notification, WDYT?

File: services/src/test/java/org/exoplatform/news/notification/plugin/SendKudosForPostedNewsNotificationPluginTest.java [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+@RunWith(PowerMockRunner.class)
+@PowerMockIgnore("javax.management.*")
+public class SendKudosForPostedNewsNotificationPluginTest {
```
Comment (aelbenney) [inline services/src/test/java/org/exoplatform/news/notification/plugin/SendKudosForPostedNewsNotificationPluginTest.java]: The test coverage ratio has reached 0.68, can you update the property in the pom file please.

File: webapp/src/main/webapp/WEB-INF/conf/news/notification/configuration.xml [hunk 3]
```diff
@@ -301,6 +301,87 @@
...
     </component-plugin>
+    <component-plugin>
+      <name>notification.plugins</name>
+      <set-method>addPlugin</set-method>
+      <type>org.exoplatform.news.notification.plugin.SendKudosForPostedNewsNotificationPlugin</type>
+      <description>Initial information for plugin SendKudosForPostedNewsNotificationPlugin</description>
+      <init-params>
+        <object-param>
+          <name>template.SendKudosForPostedNewsNotificationPlugin</name>
+          <description>The template for the plugin of the state updates of contents</description>
```
[the remark this answers is not in the document]
Comment (mohamedHedi1990) [reply webapp/src/main/webapp/WEB-INF/conf/news/notification/configuration.xml]: Fixed.

File: webapp/src/main/webapp/WEB-INF/conf/news/notification/configuration.xml [hunk 4]
```diff
@@ -301,6 +301,87 @@
...
+    </component-plugin>
+    <component-plugin>
+      <name>notification.plugins</name>
+      <set-method>addPlugin</set-method>
+      <type>org.exoplatform.news.notification.plugin.SendKudosForSharedNewsNotificationPlugin</type>
+      <description>Initial information for plugin SendKudosForSharedNewsNotificationPlugin</description>
+      <init-params>
+        <object-param>
+          <name>template.SendKudosForSharedNewsNotificationPlugin</name>
+          <description>The template for the plugin of the state updates of contents</description>
```
[the remark this answers is not in the document]
Comment (mohamedHedi1990) [reply webapp/src/main/webapp/WEB-INF/conf/news/notification/configuration.xml]: Fixed.

File: services/src/main/resources/locale/portlet/news/News_en.properties [hunk 5]
```diff
@@ -89,13 +89,17 @@ news.notification.description.like.myNews={0} liked your article "{1}" in the {2
 news.notification.description.like.mySharedNews={0} liked the article "{1}" you shared in the {2} space
 news.notification.description.comment.myNews={0} commented your article "{1}" in the {2} space
 news.notification.description.comment.mySharedNews={0} commented the article "{1}" you shared in the {2} space
+news.notification.description.sendKudos.myPostedNews={0} sent you a kudos for the article "{1}" in the {2} space
+news.notification.description.sendKudos.mySharedNews={0} sent you a kudos for the article "{1}" you shared in the {2} space
```
[the remark this answers is not in the document]
Comment (mohamedHedi1990) [reply services/src/main/resources/locale/portlet/news/News_en.properties]: The notification messages were specified in the US description.

File: webapp/src/main/webapp/WEB-INF/notification/templates/web/postNewsNotificationPlugin.gtmpl (hunk 1 above)
Comment (mohamedHedi1990) [reply webapp/src/main/webapp/WEB-INF/notification/templates/web/postNewsNotificationPlugin.gtmpl]: It was not specified.

File: services/src/test/java/org/exoplatform/news/notification/plugin/SendKudosForPostedNewsNotificationPluginTest.java (hunk 2 above)
Comment (mohamedHedi1990) [reply services/src/test/java/org/exoplatform/news/notification/plugin/SendKudosForPostedNewsNotificationPluginTest.java]: Fixed.

**Pull request closed without merging**